docs/.vitepress/config.js

```diff
@@ -34,6 +34,7 @@ export default {
           { text: 'Slider', link: '/components/slider' },
           { text: 'Text Inputs', link: '/components/text-inputs' },
           { text: 'Search Filter', link: '/components/search-filter' },
+          { text: 'Toggle', link: '/components/toggle' },
```

Comment:
I added the CSS class but I also think it would be worth just making another Vue component (similar to `Button` which can be used as the toggle)